### ISSUE TITLE
Where is the difference between creating data with or without virtual point?

### ISSUE THREAD

**Issue by zeyuwang615** [tianweiy/MVP#25]
Hi! Thank you for your great work!

I am following the instructions in getting_started. However, I fail to understand the difference between creating data with virtual points (MVP) and creating data without virtual points (CenterPoint Baseline). Based on the instructions, the command for MVP is
```
# nuScenes
python tools/create_data.py nuscenes_data_prep --root_path=NUSCENES_TRAINVAL_DATASET_ROOT --version="v1.0-trainval" --nsweeps=10 --virtual True 
```
And the command for CenterPoint is
```
# nuScenes
python tools/create_data.py nuscenes_data_prep --root_path=NUSCENES_TRAINVAL_DATASET_ROOT --version="v1.0-trainval" --nsweeps=10 --virtual False 
```

It seems to me that the only difference is `--virtual`, which is only used in creating gt sampling [here](https://github.com/tianweiy/CenterPoint/blob/db36c497a71014961c1ec17042a7524a79d4e792/tools/create_data.py#L19). But according to the instructions, the improvements of gt sampling on nuscenes is marginal and thus can be simply discarded. If that is the case, it seems to me there is no real difference between creating data with or without virtual points.

Am I missing something here? Thanks!

**Comment by tianweiy** [comment]
you are right that the only difference is whether or not we create gt dataset with virtual points.

> But according to the instructions, the improvements of gt sampling on nuscenes is marginal and thus can be simply discarded

In general, this is right. But if we want to compete with other approaches, we need this extra 0,5 nds :)
